hookman/hookman_generator.py

```diff
@@ -250,7 +254,7 @@ def _validate_package_folder(self, artifacts_dir, assets_dir):
             raise AssetsDirNotFoundError()
 
         if not artifacts_dir.exists():
-            raise ArtifactsDirNotFoundError()
+            raise ArtifactsDirNotFoundError(f'Artifacts directory not found: {artifacts_dir}')
```

Comment:
Perhaps you should update the test[ `test_generate_plugin_package_with_missing_folders`](https://github.com/ESSS/hookman/blob/146fae48fc9f28158ec2c07373f88d79816dfacf/tests/test_hookman_generator.py#L159)

Reply:
Good catch, done!